clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/input/kafka/KafkaInputDictTrainer.java

```diff
@@ -119,6 +125,9 @@ public Param build() {
   private final CompressionStrategy sourceVersionCompressionStrategy;
   private final CompressorBuilder compressorBuilder;
 
+  // For testing purpose
```

Comment:
Nitpick... it seems to me that it's the setter for this variable that is for testing purposes, but not the variable itself. Is that right?